pemicro ships pre-built proprietary shared libraries we can not distribute
---
diff -ruN a/spsdk/debuggers/__init__.py b/spsdk/debuggers/__init__.py
--- a/spsdk/debuggers/__init__.py	2022-02-04 14:27:29.000000000 +0100
+++ b/spsdk/debuggers/__init__.py	2022-02-14 00:01:52.710415054 +0100
@@ -9,5 +9,4 @@
 
 from .debug_probe import DebugProbe
 from .debug_probe_jlink import DebugProbePyLink
-from .debug_probe_pemicro import DebugProbePemicro
 from .debug_probe_pyocd import DebugProbePyOCD
diff -ruN a/spsdk/debuggers/utils.py b/spsdk/debuggers/utils.py
--- a/spsdk/debuggers/utils.py	2022-02-04 14:27:29.000000000 +0100
+++ b/spsdk/debuggers/utils.py	2022-02-14 00:05:11.017196467 +0100
@@ -15,7 +15,6 @@
 from spsdk import SPSDKError
 from spsdk.debuggers.debug_probe import DebugProbe, SPSDKDebugProbeError, SPSDKProbeNotFoundError
 from spsdk.debuggers.debug_probe_jlink import DebugProbePyLink
-from spsdk.debuggers.debug_probe_pemicro import DebugProbePemicro
 
 # Import all supported debug probe classes
 from spsdk.debuggers.debug_probe_pyocd import DebugProbePyOCD
@@ -23,7 +22,6 @@
 PROBES = {
     "pyocd": DebugProbePyOCD,
     "jlink": DebugProbePyLink,
-    "pemicro": DebugProbePemicro,
 }
 
 logger = logging.getLogger(__name__)
